LfsCompiler/CombinedConfig.cs

```diff
@@ -38,7 +38,7 @@ public CombinedConfig(IEnumerable<string> additionalInvariantMethodNames, string
             this.TranslationCompiler = new SdvTranslationCompiler(this, projectPath);
             this.gitHubUrlRoot = new Lazy<string?>(this.GetGithubBaseUrl);
             this.gitRepoRootFolder = new Lazy<string?>(() => this.ExecuteGitCommand("rev-parse --show-toplevel"));
-            this.gitHeadCommit = new Lazy<string?>(() => this.ExecuteGitCommand("rev-parse head"));
+            this.gitHeadCommit = new Lazy<string?>(() => this.ExecuteGitCommand("rev-parse HEAD"));
```

Comment:
I think what really needs to happen here is to alter it to not call `git` at all and just read .git/HEAD (Some folks have had trouble compiling because `git` wasn't on the path).  But let's not let great be the enemy of good.